### DISCUSSION TITLE
How to keep Arduino libraries up to date

### DISCUSSION THREAD

**Discussion by egonza1661** [Sloeber/arduino-eclipse-plugin#1454] [General]
Hello, I realize that Arduino libraries are newer than the ones that are installed in the plugin.
¿What do I have to do to update the libraries in the plugin?

**Comment by jantje** [comment]
goto menu item Arduino (or sloeber)-> preferences ->library manager and update the libraries you want to update.
To be sure the new code is used you can do a project clean

**Reply by egonza1661** [reply]
Sorry to bother you again.
I can't find how to update the libraries.
From here what do I do?
![imagen](https://user-images.githubusercontent.com/70854619/158223410-d793c63a-cfcd-4334-b401-1473510923b7.png)

**Comment by jantje** [comment]
click the trangle before the checkmark

**Reply by egonza1661** [reply]
Ok. I've got it. After clicking the triangle mark, I 've selected the library, and then I 've choosen the last version for that library. I've Applied changes and It's done.
Thank you very much.
By the way, your are doing a great job.
Do I have to close the issue?

**Comment by jantje** [comment]
You have created a discussion, not an issue. I don't think you can close a discussion.
I do think you can select and answer that solved your question. I'm not sure as discussions are pretty new to me.
